make.sh

```diff
@@ -110,6 +121,12 @@ if [[ "$ARCH_BUILD" != "single" ]]; then
     docker cp $container_arm64:/usr/lib/aarch64-linux-gnu/libgomp.so.1.0.0 build/arm64
 fi
 
+FROM_ALPINE="$container_alpine:/lightgbm/LightGBM"
+docker cp $FROM_ALPINE/lib_lightgbm.so build/alpine
```

Comment:
The README needs to be updated to account for this new build/alpine. And probably this new build/alpine should be inside the amd64 folder. Please check with other members the best structure having in mind that changing the directory structure may be a breaking change and will require a major release.

Reply:
I moved the alpine build into the amd64 folder. So, the new structure is: build/amd64/glibc, build/amd64/alpine and build/arm64. I will also update the README in the next commit.